tooling/ci/install-required-packages-for-integration.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#
+#
+# Install all the necessary packages for spinning up a server
+#
+
+. ~/.bashrc
+
+sudo apt-get install libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
```

Comment:
Curious why we have to install all this stuff manually if we're using a Docker image specifically for Cypress?

Reply:
This I actually didn't find much reference. If I don't have these added, ci complains about it. Theoretically it should have been included by orb. I guess I could dig more into it, but not sure whether it'd be worth the effort..

Reply:
Definitely not worth blocking this